build/Targets/GenerateAssemblyInfo.targets

```diff
@@ -19,6 +19,9 @@
     <AssemblyVersionAttribute Include="System.Reflection.AssemblyInformationalVersionAttribute">
       <_Parameter1>$(NuGetPerBuildPreReleaseVersion)</_Parameter1>
     </AssemblyVersionAttribute>
+    <AssemblyVersionAttribute Include="System.Reflection.AssemblyDescriptionAttribute">
```

Comment:
So after @tannergooding's change, we're putting all sorts of text in there. So the docs and practice seem to be disagreeing already.

The divisional build puts branch name in _some_ field, any idea which one that is?